sdk/src/main/java/com/akkaserverless/javasdk/JsonSupport.java

```diff
@@ -0,0 +1,118 @@
+/*
+ * Copyright 2021 Lightbend Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.akkaserverless.javasdk;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.protobuf.Any;
+import com.google.protobuf.ByteString;
+import com.google.protobuf.UnsafeByteOperations;
+
+import java.io.IOException;
+import java.util.Optional;
+
+public final class JsonSupport {
+
+  public static final String AKKA_SERVERLESS_JSON = "json.akkaserverless.com/";
+
+  private static final ObjectMapper objectMapper = new ObjectMapper();
```

Comment:
oh, we should probably have a better configured ObjectMapper by default. Maybe even use some subset of Akka's configuration mechanism for Jackson. I'll create an issue.